OpenRA.Mods.Common/Widgets/Logic/Installation/InstallFromDiscLogic.cs

```diff
@@ -493,15 +493,6 @@ string FindSourcePath(ModContent.ModSource source, IEnumerable<string> volumes)
 				if (Platform.CurrentPlatform != PlatformType.Windows)
 					return null;
 
-				foreach (var prefix in source.RegistryPrefixes)
```

Comment:
We'll need to add this back!

Reply:
The question is how we should handle on Core.

```cs
var path = Microsoft.Win32.Registry.GetValue(prefix + source.RegistryKey, source.RegistryValue, null) as string;
if (path == null)
     continue;
```